src/main/kotlin/io/bazel/kotlin/builder/JdepsMergerComponent.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2018 The Bazel Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package io.bazel.kotlin.builder;
+
+import dagger.Component;
+import dagger.Provides;
+import io.bazel.kotlin.builder.tasks.BazelWorker;
+import io.bazel.kotlin.builder.tasks.jvm.JdepsMerger;
+import java.io.PrintStream;
+import javax.inject.Singleton;
+
+@Singleton
+@dagger.Component(modules = {JdepsMergerComponent.Module.class})
+public interface JdepsMergerComponent {
+
+  BazelWorker worker();
+
+  @Component.Builder
+  interface Builder {
+    JdepsMergerComponent build();
+  }
+
+  @dagger.Module
+  class Module {
+
+    @Provides
+    public PrintStream provideDebugPrintStream() {
```

Comment:
Super-nit: Technically these can all be static methods and the module declared abstract. This avoids creating an instance.

Reply:
Done